src/coreclr/jit/gentree.cpp

```diff
@@ -6957,8 +6957,9 @@ GenTreeIntCon* Compiler::gtNewStringLiteralLength(GenTreeStrCon* node)
         return gtNewIconNode(0);
     }
 
-    int             length = -1;
-    const char16_t* str    = info.compCompHnd->getStringLiteral(node->gtScpHnd, node->gtSconCPX, &length);
+    constexpr int maxStrSize      = 1024;
+    char16_t      str[maxStrSize] = {};
```

Comment:
Also, you do not really need the string at all, so the buffer can just be `null` in this case.

Reply:
Removed and made buffer nullptr, since it's doesn't add a lot of value to dump it